### PR TITLE
Features

### PR DESCRIPTION
**Pull request by SigmaSam**
Weather cast site:
- Search Bar that allows you to check for any city.
- Returns to you the temperature and current climate of the city and also a small Wikipedia description from the city using API's.
- Toggle switch that allows you to swap the temperature between °C and °F.

### CODE REVIEW

File: README.md
Comment (peterrobert) [inline README.md]: When I try clicking on the live version, it seems the link is not properly working since it's returning an empty page. ⬇️ 

- [ ] Kindly fix the live version. This will make your project more professional. Kindly you can use this [link](https://raw.githack.com/) to properly set it up.

File: src/index.js
Comment (peterrobert) [inline src/index.js:9]: - [ ] Kindly separate the dom-manipulation tasks from the application logic. This will keep your code clean and allow you to create different files for different tasks which is always good practice.  Since you have already learned about modules in javascript, kindly use that power to separate your code. ⬇️ 
dom_logic_separation_requirement

File: package.json
Comment (peterrobert) [inline package.json]: Great work adding web-pack to your project. 👏🏿 

- [ ] Kindly ensure all web pack-related packages are added to the package.json. This will ensure that proper configuration is in place. It seems you missed one. Kindly you should also add `webpack-dev-server` ⬇️ 

devdep_requirement (1)

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED ♻️ 🛑 
Hello, @SigmaSam  👋🏿 very nice project. The project works as expected. 👏🏿 Great work trying to implement all the requirements. This was a great effort. However, there are changes to be made kindly check below ⬇️ 

- [x] The application should change the appearance based on the received data. This is a hard requirement for this project and also it will greatly improve the UI.  You can do this by maybe changing the icons based on the received data for example rainy, sunny, etc. ⬇️ 
appearance_requirement

- [ ] Commit messages should always start with a capital letter and they should in the right tense. This will make your project even more professional. For example instead of updated linters, It should be Update linters.


### [OPTIONAL]

- [ ] Kindly add a loading component that displays after the request is made till the response is returned. This will improve your UI.
- [ ] Kindly add functionality that when a user first loads the page, it gets his geological location and returns the data of his specific location. To achieve this you can use HTML5 geolocation API. Kindly read this [article](https://www.w3schools.com/html/html5_geolocation.asp) for more on this.

### if you have any question kindly reach me via slack @peterrobert

### HAPPY CODING 💻

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @SigmaSam :wave:

__Thank you for implementing most of the changes required by the previous reviewer.
However, you need to fix some other issues before the project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] As mentioned by the last reviewer, your live demo link seems to be broken. Are you having challenges with githack? Netlify is another alternative.
- [ ] I couldn't run the `npm start` script because the required dependency (`webpack-dev-server`) is missing. Kindly add it to the devDependencies section of your `package.json` file.




* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work. 💻 👀 ✨ 
- Good job adding the requested dependencies.

## However, there are some changes required 👀 🧰 🔨 🔧 
- The Live Demo link on your README file has not been updated and is still not working. Please fix it as the previous TSE requested.
- There is a broken image on your README. Please fix this as well.

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED✔✔✔✔**
Great project and correction of reviews.
You can merge👍👍👍

**Pull request merged**
